Dockerfile

```diff
@@ -44,6 +44,7 @@ COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 
 
 RUN python manage.py collectstatic --noinput
+RUN python manage.py migrate
```

Comment:
I don't think the migrate should run as part of the docker build, should it?

I would expect this to happen as part of the kubernetes deployment, because we will be moving away from sqlite, and we won't want the database to be baked into the image. Once we externalise the db, this will be environment dependent and we won't have access to the right credentials at build time.

Reply:
Agreed this will live outside once we utilise an external database.